src/Sai2Model.h

```diff
@@ -726,6 +727,29 @@ class Sai2Model {
 	void displayJoints();
 	void displayLinks();
 
+	/**
+	 * @brief Add load to link. This is done by adding a fixed link with load 
+	 * parameters to the existing link.
+	 * 
+	 * @param link_name 	link to add load
+	 * @param mass 			load mass
+	 * @param com_pos 		load com measured about the parent joint frame
+	 * @param inertia 		load inertia measured about the parent joint frame 
+	 * @param body_name 	unique body name for identifier
+	 */
+	void addLoad(const std::string& link_name,
```

Comment:
Can we use a Transform3d to set the position and orientation of the fixed body in parent instead of just the position ? That way we can use the inseria of the added body in its own inertial frame which is easier to use in general